### PR TITLE
docs: Waypoint unification work (dev-portal)

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR is for the Waypoint unification work. 

Asana [task](https://app.asana.com/0/1201979547919485/1201981221986512)

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
The end result of the unification is one learn tutorial that will be deprecated. The input variables [tutorial](https://learn.hashicorp.com/tutorials/waypoint/input-variables?in=waypoint/configuration). Right now the content is duplicate with the doc page.

**Review by BrandonRomano — APPROVED** [review]
Has my approval by way of it building successfully / everything looking in good working order... Thanks @karl-cardenas-coding 💯 

I'll defer the content review to @tunzor / @topfunky.

File: website/content/docs/getting-started/index.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Docker
+
+You can deploy containerized workloads through Docker and Waypoint. Start deploying containerized applications by checking out the [Getting Started with Docker](/tutorials/get-started-docker/get-started-intro) tutorials.
+
+
+## Next Steps
+
+Afterwards, go in-depth with tutorials for AWS, Azure, GCP, Kubernetes, and
+Nomad [tutorials](https://learn.hashicorp.com/waypoint). For detailed
+Kubernetes-focused documentation, see the
```
Comment (tunzor) [inline website/content/docs/getting-started/index.mdx:36]: You were going to drop the "detailed kubernetes-focused documentation" line right?

**Review by tunzor — APPROVED** [review]

**Review by topfunky — CHANGES_REQUESTED** [review]
Originally the one page quick start was a key part of the onboarding and it still receives `1,000` pageviews a month, which is about 3% of all Waypoint traffic.

I talked to Mitchell and he was in favor of keeping a five minute quick start of some kind, then sending people to the full tutorial after. Could we keep the one page quick start in here somewhere?

Screen Shot 2022-04-14 at 11 06 52 AM

**Comment by karl-cardenas-coding** [comment]
> Originally the one page quick start was a key part of the onboarding and it still receives `1,000` pageviews a month, which is about 3% of all Waypoint traffic.
> 
> I talked to Mitchell and he was in favor of keeping a five minute quick start of some kind, then sending people to the full tutorial after. Could we keep the one page quick start in here somewhere?
> 
> Screen Shot 2022-04-14 at 11 06 52 AM

I can keep the existing one, and add the other platforms as links. The thing I don't like about the existing one is that it assumes the user wants to get started with a local install and Docker.

**Pull request merged**